packages/shared/src/services/__tests__/ClaudeStreamingService.test.ts

```
@@ -0,0 +1,698 @@
+            
+            // Add user message
+            yield* Effect.gen(function* () {
+              const userMsg: StreamMessage = {
+                id: `msg-${Date.now()}-user`,
+                type: "user",
+                content: userMessage,
+                timestamp: Date.now(),
+                sessionId
+              };
+              session.messages.push(userMsg);
+            });
```

Biome (2.1.2)

[error] This generator function doesn't contain yield.



(lint/correctness/useYield)